patroni/api.py

```diff
@@ -398,6 +398,7 @@ def get_postgresql_status(self, retry=False):
                     'received_location': row[3],
                     'replayed_location': row[4],
                     'replayed_timestamp': row[5],
+                    'is_lagging': self.server.patroni.ha.is_lagging(xlog=row[3]),
```

Comment:
Acceptance tests are failing on this line:
```python
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 625, in process_request_thread                                                        
    self.finish_request(request, client_address)                                                                                        
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)                                                                             
  File "/usr/lib/python3.5/socketserver.py", line 681, in __init__                                                                      
    self.handle()
  File "/usr/lib/python3.5/http/server.py", line 422, in handle                                                                         
    self.handle_one_request()
  File "/usr/lib/python3.5/http/server.py", line 410, in handle_one_request                                                             
    method()
  File "/home/akukushkin/git/ants/patroni/patroni/api.py", line 109, in do_GET_patroni
    response = self.get_postgresql_status(True)                                                                                         
  File "/home/akukushkin/git/ants/patroni/patroni/api.py", line 402, in get_postgresql_status                                           
    'paused': row[6]} if row[1] else {
TypeError: is_lagging() got an unexpected keyword argument 'xlog' 
```

I think 'is_lagging' should not be exposed via API on the first place, because:
 1. It could be calculated on any node from `replayed_location`
 2. It is using "cached" version of the cluster view in order to calculate this boolean value, what can give a wrong result, because at this moment master can already write some other value into `optime/leader`. The result will be `False`, what is wrong...